.github/workflows/publish_crates.yml

```diff
@@ -51,6 +60,8 @@ jobs:
           override: true
 
       - uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.inputs.commit || 'master' }}
```

Comment:
I'm not sure we want to default to `master`. We'd want to default to the branch that this workflow is being run against. I think that's the behavior if left empty.

```
    # The branch, tag or SHA to checkout. When checking out the repository that
    # triggered a workflow, this defaults to the reference or SHA for that event.
    # Otherwise, uses the default branch.
    ref: ''
```
https://github.com/actions/checkout/blob/755da8c3cf115ac066823e79a1e1788f8940201b/README.md?plain=1#L28-L31

Reply:
removed it